src/utils/vc/crypto/Ed25519VerificationKey2020.js

```diff
@@ -0,0 +1,29 @@
+import b58 from 'bs58';
+import * as base64 from '@juanelas/base64';
+import { Ed25519VerKeyName, Ed255192020VerKeyName } from './constants';
+import Ed25519VerificationKey2018 from './Ed25519VerificationKey2018';
+
+export default class Ed25519VerificationKey2020 extends Ed25519VerificationKey2018 {
+  /**
+   * Construct the public key object from the verification method
+   * @param verificationMethod
+   * @returns {Ed25519VerificationKey2020}
+   */
+  static from(verificationMethod) {
+    const isEd25519Type = verificationMethod.type.indexOf(Ed255192020VerKeyName) !== -1
+      || verificationMethod.type.indexOf(Ed25519VerKeyName) !== -1;
+    if (!verificationMethod.type || !isEd25519Type) {
+      throw new Error(`verification method should have type ${Ed255192020VerKeyName} - got: ${verificationMethod.type}`);
+    }
+
+    if (verificationMethod.publicKeyBase58) {
+      return new this(b58.decode(verificationMethod.publicKeyBase58));
+    }
+
+    if (verificationMethod.publicKeyBase64) {
+      return new this(base64.decode(verificationMethod.publicKeyBase64));
+    }
+
+    throw new Error(`Unsupported signature encoding for ${Ed255192020VerKeyName}`);
+  }
+}
```

Comment:
`verifier` function needs to be overwritten as well.

Reply:
it doesn't seem to need to be in order to work, it behaves same as ed25519 2018

Reply:
Do you expect people reading this code to know this? Why not keep it clear?

Reply:
it seemed pretty obvious given it extends Ed25519VerificationKey2018 while all others dont typically extend from another defined class

Reply:
have updated with a comment at the end of the class